docs/contributorfile.rst

```diff
@@ -172,11 +182,25 @@ At physiopy, we follow a very similar workflow. The only two differences are:
 
 Pull Requests
 -------------
-To improve understanding pull requests "at a glance", we use the same labels used for issues. Multiple labels can be assigned - just think which ones suit your PR the most! 
+To improve understanding pull requests "at a glance", we use the labels listed above. Multiple labels can be assigned, but it's a good idea to keep different types of contributions separated, unless they are minimal - for instance, you might want to open a PR for code, one for documentation and one for testing.
+In general, if you're tempted to assign more than one label that would trigger a release, you might want to split your PR instead.
 When opening a pull request, assign it to at least one label.
 
 We encourage you to open a PR as soon as possible - even before you finish working on them. This is useful especially to you - so that you can receive comments and suggestions early on, rather than having to process a lot of comments in the final review step! 
 However, if it’s an incomplete PR, please open a **Draft PR**. That helps us process PRs by knowing which one to have a look first - and how picky to be when doing so.
+
+Reviewing PRs is a time consuming task, that can be stressful for both the reviewer and the author. Avoiding wasting time and the need of little fixes - such as fixing grammar mistakes and typos, styling code, or adopting conventions - is a good start for a successful (and quick) review. Before graduating a Draft PR to a PR ready for review, please check that:
+
+- You did all you wanted to include in your PR. If at a later stage you realise something is missing and it's not minor things, you will need to open a new PR
+- Your code is harmonious with the rest of the code - no repetitions of any sort!
+- Your code respects the `adopted Style <#styling>`_, especially:
+    - your code is lintered adequately and respects the `PEP8 <https://www.python.org/dev/peps/pep-0008/>`_ convention
```

Comment:
I'd mention flake8 here.

Reply:
I mention it in the style guide already, do you think it is necessary to repeat? Or are you proposing to delete the Style guide?

Reply:
Oh okay, no need to mention it then!